src/main/java/com/shurjomukhi/model/Transaction.java

```diff
@@ -0,0 +1,113 @@
+package com.shurjomukhi.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.shurjomukhi.ShurjopayStatus;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * After verifying payment response

```

Comment:
why this new model ?

Reply:
This new model came from "[a5108aa]" this commit. No need this model. Now it deleted.